Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -485,6 +487,7 @@ const DEFAULT_VALUES = {
   // _currentUpdateSelectionCallID: null,
   viewId: null,
   widgets: [],
+  activeWidget: null,
```

Comment:
we typically do not declare in DEFAULT_VALUES parameters that are null

Reply:
I've typed `getActiveWidget()` as `Nullable<vtkAbstractWidget>`, so I'd like to initialize `activeWidget` on construction to `null` rather than the implicit `undefined`. Would it be better to instead do `model.activeWidget = null` in the constructor function above? (Note that the current null assignment line is inside `updateSelection()`.)

Reply:
What you say makes sense to me. 

Shall we add `undefined` to the `Nullable` type. (probably not a good idea).

@jourdain a suggestion ?

Reply:
I'd avoid adding undefined to `Nullable` b/c `null !== undefined`, even though `null == undefined`.

Reply:
I think that is fine especially with TS definition.